unstructured/staging/base.py

```diff
@@ -161,7 +161,7 @@ def convert_to_isd_csv(elements: List[Element]) -> str:
 
         if row.get("sent_from"):
             row["sender"] = row.get("sent_from")
-            if type(row["sender"]) == list:
+            if isinstance(type(row["sender"]), list):
```

Comment:
There are a couple of files with pure formatting changes like this that shouldn't be included in this PR.

Reply:
I added these because checks were failing. Should I move these to a different PR and have that merged before merging this one?

Reply:
Pulling in the changes from #986 (once that's merged) will take care of this.